_posts/2018-01-02-in-defence-of-swap.md

```diff
@@ -196,7 +196,7 @@ v2](https://www.youtube.com/watch?v=ikZ8_mRotT4).
 ### Under moderate/high memory contention
 
 - **With swap:** All memory types have an equal possibility of being reclaimed.
-  This means we have more chance of being able to reclaim pages successfully --
+  This means we have more chances of being able to reclaim pages successfully --
```

Comment:
this one is actually correct :-) if it was "chances" it would have to be "chances to", but "chance of" is fine